packages/devtools/tests/__snapshots__/launcher.test.js.snap

```diff
@@ -107,16 +107,19 @@ Array [
         "--disable-default-apps",
         "--mute-audio",
         "--no-first-run",
+        "--no-default-browser-check",
         "--disable-hang-monitor",
         "--disable-prompt-on-repost",
         "--disable-client-side-phishing-detection",
+        "--disable-features=TranslateUI",
         "--password-store=basic",
         "--use-mock-keychain",
         "--disable-component-extensions-with-background-pages",
         "--disable-breakpad",
         "--disable-dev-shm-usage",
         "--disable-ipc-flooding-protection",
         "--disable-renderer-backgrounding",
+        "--force-fieldtrials=*BackgroundTracing/default/",
```

Comment:
Do we need to disable background tracing by default? Will this have any impact on trace gatherer? @christian-bromann

Reply:
From looking at the [implementation in `chrome-launcher`](https://github.com/GoogleChrome/chrome-launcher/blob/v0.13.4/src/chrome-launcher.ts#L156-L174) it looks to me like it will always use `DEFAULT_FLAGS` if `ignoreDefaultFlags` is falsy. `ignoreDefaultFlags` should always have been falsy, since we weren't passing it (we passed `ignoreDefaultArgs`). [`DEFAULT_FLAGS`](https://github.com/GoogleChrome/chrome-launcher/blob/v0.13.4/src/flags.ts#L8) contains `--force-fieldtrials=*BackgroundTracing/default/`. So it seems to me like we've always gotten the argument, so this PR shouldn't have any impact on this, as from what I can tell, nothing changes in this regard. Of course, if you'd like to actually stop using the argument, let me know!